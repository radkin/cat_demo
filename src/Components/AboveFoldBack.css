.kittyRow {
  padding-top: 405px;
  display:flex;
  flex-direction:row;
  justify-content: flex-start;
  margin: auto;
  width: 50%
}

.graph {
  padding-left: 10px;
  width: 100%;
}

.pic {
  border-radius: 20px;
  height: 90%;
  width: 90%;
}

p {
  color: white;
  font-family: sans-serif;
}
