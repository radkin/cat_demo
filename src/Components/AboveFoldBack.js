import React, { Component } from 'react';
import Fade from 'react-reveal/Fade';
import rrConfig from 'react-reveal/globals';
import './AboveFoldBack.css';
import gusPerch from '../assets/images/gus_perch_sm.jpg';
import sleepKit from '../assets/images/sleeping_kitties_sm.jpg';

rrConfig({ ssrFadeout: true });

class AboveFoldBack extends Component {
  render() {
    return(
      <div className="kittyRow">

        <div className="kitty">
          <Fade left>
            <p>They may be cute when they're sleeping</p>
            <p>but these beasties will wake you up in</p>
            <p>the middle of the night!</p>
          </Fade>
        </div>
        <div className="kitty">
          <Fade left delay={100}>
            <img src={gusPerch} alt="gus perch" className="pic"/>
          </Fade>
        </div>

        <div className="kitty">
          <Fade left delay={200}>
            <p>They may be cute when they're sleeping</p>
            <p>but these beasties will wake you up in</p>
            <p>the middle of the night!</p>
          </Fade>
        </div>
        <div className="kitty">
          <Fade left delay={300}>
            <img src={sleepKit} alt="sleeping kitties" className="pic"/>
          </Fade>
        </div>

      </div>
    )
  }
}

export default AboveFoldBack;
