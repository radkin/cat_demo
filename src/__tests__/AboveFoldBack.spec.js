import React from 'react';
import renderer from 'react-test-renderer';
import { mount } from 'enzyme';

import AboveFoldBack, { Counter, dataReducer } from '../Components/AboveFoldBack';

describe('AboveFoldBack', () => {

  test('snapshot renders', () => {
    const component = renderer.create(<AboveFoldBack />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

});
