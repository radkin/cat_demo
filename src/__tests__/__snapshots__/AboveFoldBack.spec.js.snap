// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboveFoldBack snapshot renders 1`] = `
<div>
  <div
    className="graph"
  >
    <div
      className="react-reveal text"
      style={
        Object {
          "opacity": undefined,
        }
      }
    >
      Predict cellular growth and chart nerve regeneration with incredible precision.
    </div>
  </div>
  <div
    className="graph"
  >
    <img
      alt="graph2"
      className="react-reveal pic1"
      src="graph2.svg"
      style={
        Object {
          "opacity": undefined,
        }
      }
    />
  </div>
  <div
    className="graph"
  >
    <div
      className="react-reveal text"
      style={
        Object {
          "opacity": undefined,
        }
      }
    >
      Using Skyline Architecture you will be able to see alternate dimensions as if peering through a mirror.
    </div>
  </div>
  <div
    className="graph"
  >
    <img
      alt="graph1"
      className="react-reveal pic2"
      src="graph1.svg"
      style={
        Object {
          "opacity": undefined,
        }
      }
    />
  </div>
</div>
`;
